`make docker` is broken on master

## Bug Report

<!-- Thanks for your bug report! Don't worry if you can't fill out all the sections. -->

### What version of TiKV are you using?
<!-- You can run `tikv-server --version` -->
https://github.com/tikv/tikv/commit/079a806e549f25dbf86b75bdc5087f667a713476
https://github.com/tikv/tikv/commit/b67e395c09bd2b2a6b863b030fa05f0771918221

### What operating system and CPU are you using?
<!-- If you're using Linux, you can run `cat /proc/cpuinfo` -->
centos7

### Steps to reproduce
<!-- If possible, provide a recipe for reproducing the error. A complete runnable program is good. -->
`make docker`

### What did you expect?
`make docker` succeed.

### What did happened?
`make docker` failed with the following error:
```
19:31:27  [0mThe command '/bin/sh -c mkdir -p ./cmd/src/bin &&     echo 'fn main() {}' > ./cmd/src/bin/tikv-ctl.rs &&     echo 'fn main() {}' > ./cmd/src/bin/tikv-server.rs &&     for cargotoml in $(find . -name "Cargo.toml"); do         sed -i '/fuzz/d' ${cargotoml} &&         sed -i '/test\_/d' ${cargotoml} &&         sed -i '/profiler/d' ${cargotoml} &&         sed -i '/\"tests\",/d' ${cargotoml} ;     done &&     make build_dist_release' returned a non-zero code: 2
19:31:27  make: *** [docker] Error 2
```

--- a/components/batch-system/Cargo.toml
+++ b/components/batch-system/Cargo.toml
@@ -24,10 +24,12 @@ required-features = ["test-runner"]
 
 [[bench]]
 name = "router"
+path = "benches/router.rs"
 harness = false
 required-features = ["test-runner"]
 
 [[bench]]
 name = "batch-system"
+path = "benches/batch-system.rs"
 harness = false
 required-features = ["test-runner"]

